x-pack/plugins/enterprise_search/public/applications/app_search/components/api_logs/utils.ts

```diff
@@ -10,3 +10,12 @@ export const getDateString = (offSetDays?: number) => {
   if (offSetDays) date.setDate(date.getDate() + offSetDays);
   return date.toISOString();
 };
+
+export const getStatusColor = (status: number) => {
+  let color = '';
+  if (status >= 100 && status < 300) color = 'secondary';
+  if (status >= 300 && status < 400) color = 'primary';
+  if (status >= 400 && status < 500) color = 'warning';
+  if (status >= 500) color = 'danger';
+  return color;
+};
```

Comment:
I basically just copied this over as-is from the standalone UI. Def. open to thoughts or suggestions if you have any changes you'd like to see

Reply:
Fine with me